src/actions.cpp

```diff
@@ -593,7 +596,8 @@ namespace Action {
         {
 #if   defined(EXV_HAVE_REGEX)
             std::smatch m;
-            result = std::regex_search(key,m, *g);
+            std::regex reg;
```

Comment:
*g should fill std::regex object but I've no idea how do I convert it. This problem occurs in more places